db/migrate/20230127085305_create_comment.rb

```diff
@@ -1,15 +1,15 @@
 class CreateComment < ActiveRecord::Migration[7.0]
   def change
     create_table :comments do |t|
-      t.integer :author_id
-      t.integer :post_id
+      t.integer :AuthorId
+      t.integer :PostId
       t.text :Text
 
       t.timestamps
     end
-    add_foreign_key :comments, :users, column: :author_id, primary_key: :id
-    add_index :comments, :author_id
-    add_foreign_key :comments, :posts, column: :post_id, primary_key: :id
-    add_index :comments, :post_id
+    add_foreign_key :comments, :user, column: :AuthorId, primary_key: :id
```

Comment:
- [ ] Please make sure the relation between your tables is correct, the migration can't be complete which means I can't run the test to certify if all are passing. Please kindly fix that 🙃 🙃 

<img width="1032" alt="Screen Shot 2023-02-01 at 13 00 43" src="https://user-images.githubusercontent.com/74501523/216015735-1ec13ef8-78ff-4be4-a119-87c212c29a30.png">